llvm/test/Transforms/LoopStrengthReduce/X86/pr76504.ll

```diff
@@ -0,0 +1,30 @@
+; Reduced from https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=65323 by @RKSimon
+;
+; RUN: opt -S -passes=loop-reduce %s | FileCheck %s
+;
+; Make sure we don't trigger an assertion.
```

Comment:
In the future please use filename that are more expressive.
You can still reference the bug number in the comment.